docs/getting-started.rst

```diff
@@ -152,35 +160,97 @@ You can now develop Python Pulumi code in that directory, referring to this modu
 
   from tb_pulumi import (ec2, fargate, secrets)
 
-
-Use this module
-^^^^^^^^^^^^^^^
-
-When you issue ``pulumi`` commands (like "up" and "preview" and so on), it looks for a ``__main__.py`` file in your
-current directory and executes the code in that file. To use this module, you'll import it into that file and write up
-some code and configuration files.
-
-
 Create a config file
 """"""""""""""""""""
 
-It is assumed that a config file will exist at ``config.$STACK.yaml`` where ``$STACK`` is the currently selected Pulumi
-stack. This file must contain a mapping of names of config settings to their desired values. Currently, only one such
-setting is formally recognized. That is ``resources``.
-
-This is a mostly arbitary mapping that you will have to interpret on your own. This allows for flexibility, but we
-recommend some conventions here. Namely:
-
-* ``resources`` should be a mapping where the keys are the Pulumi type-strings for the resources they are configuring.
-  For example, if you want to build a VPC with several subnets, you might use the ``tb_pulumi.network.MultiCidrVpc``
-  class. Following this convention, that should be accompanied by a ``tb:network:MultiCidrVpc`` key in this mapping.
-* The values these keys map to should themselves be mappings. This provides a convention where more than one of each
-  pattern are configurable. The keys here should be arbitrary but unique identifiers for the resources being configured.
-  F/ex: ``backend`` or ``api``.
-* The values these keys map to should be a mapping where each key/value combo is a valid configuration option for the
-  resources being built. The full listing of these values can be found by browsing the :py:mod:`tb_pulumi` 
-  documentation. A barebones example can be found in our `sample config
-  <https://github.com/thunderbird/pulumi/blob/main/config.stack.yaml.example>`_.
+Create a config file for each stack, i.e., ``config.$STACK.yaml`` (where ``$STACK`` is the currently selected Pulumi
+stack). This file maps config settings to their desired values. Currently, only the ``resources``
+setting is formally recognized.
+
+Let's look at an example configuration file.
```

Comment:
A concrete example might be a better way to illustrate the conventions. I found that part of the text wordy and hard to visualize what was meant.